include/envoy/router/router.h

```diff
@@ -32,6 +32,51 @@ class RedirectEntry {
 };
 
 /**
+ * CorsPolicy for Route and VirtualHost.
+ */
+class CorsPolicy {
+public:
+  virtual ~CorsPolicy() {}
+
+  /**
+   * @return std::string access-control-allow-origin value.
```

Comment:
nit: update return type. If this is a list should it be called `allowOrigins()` ?

Reply:
While I agree 100% it should be `allowOrigins` in general... since I named all vars after the header I am not sure.. ^^

Reply:
Sure no problem.

Reply:
Sure no problem.